src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Vendor new SDK and bump API version to 2023-09-02-preview.
 
+0.5.169
++++++++
+* Update "VirtualMachines" agent pool type as Public Preview feature.
+
```

Comment:
Recently, there have been many PRs for AKS. Could you collaborate with each other to release these features in the same version to avoid conflicts and release multiple extension versions in a short period of time.

Reply:
I will put the change to Pending, then it can released together